### DISCUSSION TITLE
Add MTet Translation to Auto Translate

### DISCUSSION THREAD

**Discussion by zvn00** [SubtitleEdit/subtitleedit#7779] [Ideas]
Hi. I'm a Vietnamese translator and using Subtitle Edit everyday. Could you please consider adding MTet Translation - an AI project to the auto translate section? Thank you very much.

Here are their pages:
https://huggingface.co/spaces/VietAI/En2Vi-Translation
https://huggingface.co/VietAI/envit5-translation

And another "sister" project if needed:
https://huggingface.co/vinai/vinai-translate-en2vi-v2
https://huggingface.co/spaces/vinai/VinAI_Translate

**Comment by vcstack** [comment]
tự code đi ba, đòi hỏi

**Comment by niksedk** [comment]
Do they have a command line version?

https://github.com/vietai/mTet
